app/src/main/java/de/westnordost/streetcomplete/quests/ferry/AddFerryAccessMotorVehicle.kt

```diff
@@ -0,0 +1,31 @@
+package de.westnordost.streetcomplete.quests.ferry
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment
+
+
+
+class AddFerryAccessMotorVehicle(o: OverpassMapDataDao) : SimpleOverpassQuestType<Boolean>(o) {
+
+    override val tagFilters = "ways, relations with route=ferry and !motor_vehicle"
+    override val commitMessage = "Specify ferry access"
+    override val icon = R.drawable.ic_quest_ferry
+    override val hasMarkersAtEnds = true
+
+    override fun getTitle(tags: Map<String, String>): Int {
+        val hasName = tags.containsKey("name")
+        return if (hasName)
+            R.string.quest_ferry_motor_vehicle_name_title
+        else
+            R.string.quest_ferry_motor_vehicle_title
+    }
```

Comment:
FYI you can also write the above as this in Kotlin. I don't mind either way though
```kotlin
    override fun getTitle(tags: Map<String, String>) =
        if (tags.containsKey("name"))   R.string.quest_ferry_motor_vehicle_name_title
        else                            R.string.quest_ferry_motor_vehicle_title
```

Reply:
I do not feel that it is more readable. Rubocop linter keeps trying to use similar structure in Ruby, and I am also not fan of it.

But thanks for an info!